File: PHP/EX01.php
<?php
    
    function media($cont){
        $num = 0;
        $som = 0;
        $i = 0;
        while($i < $cont){
            echo "Digite um número:";
            $num = fgets(STDIN);
            $som = $som + intval($num);
            $i++;
        }

        $med = $som / $cont;

        return  "A media dos valores é $med";
    }

   $m = media(4);

   echo $m;

?>

File: C/EX01.c
#include <stdio.h>

int main(){
    float media;
    int sum,num,cont;

    sum = 0;
    cont = 0;

    while(cont < 20){
        printf("Digite um valor:");
        scanf("%d",&num);
        sum = sum + num;
        cont = cont + 1;
    }

    media = sum / cont;
    printf("A media dos valores é %2.f \n",media);

    return 0;
}
